build.gradle

```diff
@@ -109,6 +109,7 @@ dependencies {
     implementation "com.typesafe:config:${versions.typesafeConfig}"
     implementation "commons-io:commons-io:${versions.commonsIo}"
 
+    // TODO doesn't work under jdk14
```

Comment:
does this need to be documented so users are aware of this limitation? how would the failure manifest?

Reply:
i only discovered this working on my personal machine where i have jdk14 (openjdk). 

this failure manifests as a "missing dependency".

i haven't tried any jdk other than 8 and 14. i believe somewhere along the line they moved tools.jar